src/print_zir.zig

```diff
@@ -421,6 +421,7 @@ const Writer = struct {
             .elem_val,
             .array_type,
             .coerce_ptr_elem_ty,
+            .import,
```

Comment:
Careful, this is wrong!

Reply:
Ah good catch, thanks!

Reply:
Okay I understand Zir encoding now better--fixed this!

Reply:
Actually wait I think I can improve this further